src/next-steps/next-steps.module.css

```diff
@@ -8,18 +8,46 @@
 
 .task {
 	display: flex;
+	padding-right: 10px;
 	align-items: flex-start;
-	width: fit-content;
+}
+
+.taskListItem {
+	border: 1px solid var( --color-border-light );
+	background-color: var( --color-item-background );
+	border-radius: 6px;
+	padding: 10px;
+	display: flex;
+	flex-direction: row;
+}
+
+.buttonContent {
+	margin-left: auto;
 }
 
 .taskListItem:nth-child( n + 2 ) {
-	margin-top: 2rem;
+	margin-top: 1rem;
 }
 
 .taskTitle {
 	font-size: var( --font-header );
 	font-weight: 500;
 	color: var( --color-text-primary );
+	word-wrap: break-word;
+	word-break: break-all;
+}
+
+a.taskLink {
+	background-color: var( --color-white );
+	border: var( --color-primary ) 1px solid;
+	border-radius: 4px;
+	padding: 10px 20px;
+	display: inline-flex;
+	align-items: center;
+	font-size: var( --font-body );
+	color: var( --color-primary );
+	text-decoration: none;
+	font-weight: 500;
 }
 
```

Comment:
**Suggestion (should be done before merge, but not urgent):** Per the Figma component guide, it looks like the these kind outline fake-button styles should have a `hover` effect, and I think it would be good to add an `active` one too (it adds a sense of "weight" when clicking).

Our current color scheme is a bit off from the Figma ones, and we'll just go back and reconcile that later, so for now, this should work:

```css
a.taskLink:hover {
	background-color: var( --color-neutral-light );
}

a.taskLink:active {
	background-color: var( --color-neutral );
}
```